cmd/common-main.go

```diff
@@ -116,7 +116,7 @@ func handleCommonEnvVars() {
 
 	// In distributed setup users need to set ENVs always.
 	if !globalIsEnvCreds && globalIsDistXL {
-		logger.Fatal(uiErrEnvCredentialsMissingServer(nil), "Unable to start distributed server mode")
+		logger.Fatal(uiErrEnvCredentialsMissingDistributed(nil), "Unable to start distributed server mode")
```

Comment:
is this better `unable to start server in distributed mode`

Reply:
> is this better unable to start server in distributed mode

Done